packages/state_notifier/lib/state_notifier.dart

```diff
@@ -184,19 +184,32 @@ Consider checking `mounted`.
   /// Listeners cannot add other listeners.
   ///
   /// Adding and removing listeners is O(1).
-  RemoveListener addListener(Listener<T> listener) {
+  RemoveListener addListener(Listener<T> listener,
+      {bool fireImmediately = true}) {
```

Comment:
```suggestion
      {bool fireImmediately = true},) {
```

+ a run of dartmt to respect the style guide of Dart